Counterexamples/CliffordAlgebra_not_injective.lean

```diff
@@ -0,0 +1,317 @@
+      simp [Q', sq, (QuadraticForm.sum_apply), (QuadraticForm.comp_apply),
+        (QuadraticForm.sq_apply), (LinearMap.proj_apply)]
+    _ = _ := by simp_rw [← Pi.single_mul, Finset.sum_pi_single', Finset.mem_univ, if_pos]
+
+theorem Q'_zero_under_ideal (v : Fin 3 → K) (hv : v ∈ LinearMap.ker lFunc) : Q' v = 0 := by
+  rw [LinearMap.mem_ker, lFunc_apply] at hv
+  have h0 : α * β * γ * v 0 = 0 := by
+    have := congr_arg ((· * ·) (β * γ)) hv
+    simp only [MulZeroClass.mul_zero, mul_add, ← mul_assoc] at this
+    rw [mul_comm (β * γ) α, ← mul_assoc, mul_right_comm β γ β, mul_assoc β γ γ, X_sq, X_sq] at this
+    simpa only [MulZeroClass.mul_zero, MulZeroClass.zero_mul, add_zero, zero_add] using this
+  have h1 : α * β * γ * v 1 = 0 := by
+    have := congr_arg ((· * ·) (α * γ)) hv
+    simp only [MulZeroClass.mul_zero, mul_add, ← mul_assoc] at this
+    rw [mul_right_comm α γ α, mul_assoc α γ γ, mul_right_comm α γ β, X_sq, X_sq] at this
+    simpa only [MulZeroClass.mul_zero, MulZeroClass.zero_mul, add_zero, zero_add] using this
+  have h2 : α * β * γ * v 2 = 0 := by
+    have := congr_arg ((· * ·) (α * β)) hv
+    simp only [MulZeroClass.mul_zero, mul_add, ← mul_assoc] at this
+    rw [mul_right_comm α β α, mul_assoc α β β, X_sq, X_sq] at this
+    simpa only [MulZeroClass.mul_zero, MulZeroClass.zero_mul, add_zero, zero_add] using this
+  rw [Q'_apply, sq_zero_of_αβγ_mul h0, sq_zero_of_αβγ_mul h1, sq_zero_of_αβγ_mul h2, add_zero,
+    add_zero]
+
+/-- `Q'`, lifted to operate on the quotient space `L`. -/
+@[simps!]
+def Q : QuadraticForm K L :=
+  QuadraticForm.ofPolar
+    (fun x =>
+      Quotient.liftOn' x Q' fun a b h => by
+        rw [Submodule.quotientRel_r_def] at h
+        suffices Q' (a - b) = 0 by rwa [Q'_sub, sub_eq_zero] at this
+        apply Q'_zero_under_ideal (a - b) h)
+    (fun a x => by
+      induction x using Quotient.inductionOn
+      exact Q'.toFun_smul a _)
+    (by rintro ⟨x⟩ ⟨x'⟩ ⟨y⟩; exact Q'.polar_add_left x x' y)
+    (by rintro c ⟨x⟩ ⟨y⟩; exact Q'.polar_smul_left c x y)
+
+open CliffordAlgebra
+
+/-- Basis vectors in the Clifford algebra -/
+def gen (i : Fin 3) : CliffordAlgebra Q := ι Q <| Submodule.Quotient.mk (Pi.single i 1)
+
+local notation "x'" => gen 0
+local notation "y'" => gen 1
+local notation "z'" => gen 2
+
+/-- The basis vectors square to one -/
+@[simp]
+theorem gen_mul_gen (i) : gen i * gen i = 1 := by
+  dsimp only [gen]
+  simp_rw [CliffordAlgebra.ι_sq_scalar, Q_apply, ← Submodule.Quotient.mk''_eq_mk,
+    Quotient.liftOn'_mk'', Q'_apply_single, mul_one, map_one]
+
+/-- By virtue of the quotient, terms of this form are zero -/
+theorem quot_obv : α • x' - β • y' - γ • z' = 0 := by
+  dsimp only [gen]
+  simp only [← (LinearMap.map_smul), ← (LinearMap.map_sub), ← Submodule.Quotient.mk_smul, ←
+    Submodule.Quotient.mk_sub]
+  convert LinearMap.map_zero _ using 2
+  rw [Submodule.Quotient.mk_eq_zero]
+  norm_num [sub_zero, Ideal.span, Pi.single_apply]
+
+set_option maxHeartbeats 400000 in
```

Comment:
```suggestion
```
This isn't necessary for me (no idea if this is a consequence of earlier changes I made in my local version of the file)